### ISSUE TITLE
[Feature Request] Ability to specify CloudWatch Log group name for VPC Flow logs

### ISSUE THREAD

**Issue by air3ijai** [terraform-aws-modules/terraform-aws-vpc#836]
## Is your request related to a problem? Please describe.

As for now, we can specify just [`flow_log_cloudwatch_log_group_name_prefix`](https://registry.terraform.io/modules/terraform-aws-modules/vpc/aws/latest?tab=readme#input_flow_log_cloudwatch_log_group_name_prefix) and with all defaults we will get a kind of the the following name for CloudWatch Log group name:
```
/aws/vpc-flow-log/vpc-02b7411f86f26842b
```

## Describe the solution you'd like.
We should be able to specify CloudWatch Log group name(suffix) in order to be able to identify it by name.

A variable like `flow_log_cloudwatch_log_group_name = project-name` should permit us to set custom name instead of VPC ID:
```
/aws/vpc-flow-log/project-name
```

**Current code**
https://github.com/terraform-aws-modules/terraform-aws-vpc/blob/5939699c72827536971550db3eea8adad52d83b0/vpc-flow-logs.tf#L47

**Comment by bryantbiggs** [comment]
this is not a naming scheme we control - its a naming scheme controlled by AWS that we match here in order to "control" the log group

**Comment by air3ijai** [comment]
Not sure if I understand correctly - we cannot do this or we should not do this?

**Comment by bryantbiggs** [comment]
I was trying to find the documentation but I stand corrected - I guess we do have control over the log group name. A lot of other services usually dictate the name of the log group but I guess VPC flow logs allows complete control of the name

**Comment by antonbabenko** [comment]
Fixed in #847
